sherpa/models/parameter.py

```diff
@@ -153,7 +158,7 @@
 such as being twice the other parameter:
 
     >>> q = Parameter('other', 'beta', 4)
-    >>> p.val = 2 * p
```

Comment:
This is exactly want I wanted to achieve with testing the docstrings. When I read this, I was wondering "Oh, what does this do? Is that not circular p.val = 2 * p"? Then, I say that that's one the lines fixed here. We don't want to confuse users with examples that don't work, so it's good to fix them as we get to it.